app/models/todo_list.rb

```diff
@@ -2,5 +2,7 @@ class TodoList < ApplicationRecord
   belongs_to :user
   has_many :todos, dependent: :destroy
   validates :title, presence: true
+  has_many :collaborators, dependent: :destroy
+
```

Comment:
think of adding one more relation so that you can directly fetch collaborator users

Reply:
has_many :collaborations, class_name: 'Collaborator', dependent: :destroy